src/Components/Main/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import data from "../../data";
+
+class Main extends Component {
+  state = {
+    score: 0,
+    timer: 60,
+    currentImg: data[0].img,
+    currentName: data[0].name,
+    answer: ""
+  };
+
+  compare = () => {
+    const { currentName, answer, score } = this.state;
+    if (currentName === answer) {
+      const scoreAddOne = score + 1;
+      this.setState({
```

Comment:
why did you cut prevState from here? it's needed as you're changing existing state.